src/utils/postbox/postboxWrapper.ts

```diff
@@ -656,15 +656,51 @@ export class DispatchForum implements IForum {
 
     try {
       const metadataForOwner = await getMetadataForOwner(conn, wallet.publicKey!);
-      const result = metadataForOwner.map(md => ({mint: md.mint.toBase58(), name: md.data.name.replaceAll('\u0000', ''), uri: md.data.uri}));
+      const result = metadataForOwner.map(({ mint, data }) => ({
```

Comment:
what was the ui crash being caused by? is that a mis-handling of an error code? and how do we know if other unknown unknowns can be guarded against?

Reply:
Basically, what's happening here is that the code uses the user's held tokens and their metadata as an input. That metadata may not be well-formed, which could cause crashes in the frontend. This is particularly an issue because anyone can send another person a corrupted token with bad metadata that could instigate this crash I'm working right now on making this code do more error checking on metadata